packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -1,7 +1,21 @@
 import * as React from 'react';
+import { useStores } from 'Stores';
+import { Authorize } from '@deriv/api-types';
+import WalletAccount from 'Components/wallet-account';
+import './trading-hub.scss';
 
 const TradingHub = () => {
-    return <div className='trading-hub'>Trading Hub</div>;
+    const { client } = useStores();
+
+    const wallet_accounts = (client.wallet_accounts || []) as Required<Authorize>['account_list'];
+
+    return (
+        <div className='trading-hub'>
+            {wallet_accounts.map(account => {
+                return <WalletAccount key={client.loginid} account={account} />;
```

Comment:
@iman-fs The `key` here doesn’t seem right, `client.loginid` is outside of the mapping and is always the same thing, Should use a unique prop on `account` instead.

Reply:
for now I'm adding `index` as a workaround.

Reply:
@farzin-fs @balakrishna-binary please check new changes regarding above issue .